.gitignore

```diff
@@ -27,3 +27,4 @@ tmp
 .vs
 cmd/olympus/bin
 cmd/proxy/bin
```

Comment:
You could also add *.swp

Reply:
@devdinu what's a .swp file? is this related to intellij? Thanks!

Reply:
folks using vim editor, its the temporary swap files.